chronosphere/resource_slo.go

```diff
@@ -227,6 +227,28 @@ func reportingWindowsFromModel(windows []*models.DefinitionTimeWindow) []intsche
 	})
 }
 
+func burnRateDefinitionToModel(defs []intschema.SloDefinitionBurnRateAlertingConfig) []*models.DefinitionBurnRateDefinition {
```

Comment:
regarding the to/from model methods, are we certain that the server will send back a stable list in some order (or are we sorting it somewhere here to keep it stable?) so that we don't get unnecessary plans if things end up sorted differently?

Reply:
The server is going to store it and return as was persisted. We will not be changing the order in the backend.